examples/2d/mesh2d_repeated_texture.rs

```diff
@@ -0,0 +1,110 @@
+//! By default Bevy loads images to textures that clamps the image to the edges
+//! This example shows how to configure it to repeat the image instead.
+
+use bevy::{
+    audio::AudioPlugin,
+    image::{ImageAddressMode, ImageLoaderSettings, ImageSampler, ImageSamplerDescriptor},
+    math::Affine2,
+    prelude::*,
+    sprite::Anchor,
+};
+
+/// How much to move some rectangles away from the center
+const RECTANGLE_OFFSET: f32 = 250.0;
+/// Length of the sides of the rectangle
+const RECTANGLE_SIDE: f32 = 200.;
+/// How much to move the label away from the rectangle
+const LABEL_OFFSET: f32 = 25.;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins.build().disable::<AudioPlugin>())
+        .add_systems(Startup, setup)
+        .run();
+}
+
+fn setup(
+    mut commands: Commands,
+    asset_server: Res<AssetServer>,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<ColorMaterial>>,
+) {
+    // #11111: We use a duplicated image so that it can be load with and without
+    // settings
+    let image_with_default_sampler =
+        asset_server.load("textures/fantasy_ui_borders/panel-border-010.png");
+    let image_with_repeated_sampler = asset_server.load_with_settings(
+        "textures/fantasy_ui_borders/panel-border-010-repeated.png",
+        |s: &mut _| {
+            *s = ImageLoaderSettings {
+                sampler: ImageSampler::Descriptor(ImageSamplerDescriptor {
+                    // rewriting mode to repeat image,
+                    address_mode_u: ImageAddressMode::Repeat,
+                    address_mode_v: ImageAddressMode::Repeat,
+                    ..default()
+                }),
+                ..default()
+            }
+        },
+    );
+
+    // central rectangle with not repeated texture
+    commands.spawn((
+        Mesh2d(meshes.add(Rectangle::new(RECTANGLE_SIDE, RECTANGLE_SIDE))),
+        MeshMaterial2d(materials.add(ColorMaterial {
+            texture: Some(image_with_default_sampler.clone()),
+            ..default()
+        })),
+        Transform::from_translation(Vec3::ZERO),
+        Children::spawn(Spawn((
```

Comment:
cart expressed a preference for `children![]` in examples where possible in the PR text here: https://github.com/bevyengine/bevy/pull/17521